.changeset/famous-bees-juggle.md

```diff
@@ -0,0 +1,6 @@
+---
+"@khanacademy/math-input": patch
+"@khanacademy/perseus-core": patch
+---
+
+on keypad opened CEDAR event integration
```

Comment:
Sorry, missed this originally. These notes are what consumers of the library will review when upgrading. I think it's useful to be slightly more descriptive. Also, let's remove the reference to CEDAR here too. What do you think about:

```
Add keypad opened and closed analytics events
```